validator/src/main/resources/expected-data-template/otelFargateExpectedMetric.mustache

```diff
@@ -0,0 +1,234 @@
+-
```

Comment:
Change this file name. Fargate is generic. Let's call it, EKSFargateContaienrInsightsExpectedMetric.